src/KeyValue/KeyValue.cs

```diff
@@ -0,0 +1,64 @@
+using System.Text;
+
+namespace Spin.SDK.KeyValue;
+
+public class Store
+{
+    private readonly SpinHttpWorld.wit.imports.fermyon.spin.v2_0_0.IKeyValue.Store _store;
```

Comment:
Would it make sense to add a `using SpinHttpWorld.wit.imports.fermyon.spin.v2_0_0;` to this file so we don't have to use the fully-qualified type names?

Reply:
I changed this to have the following using statement

```csharp
using SpinHttpWorld.wit.imports.fermyon.spin.v2_0_0;
```

This is due to increased readability of higher level `Store` vs generated `IKeyValue.Store`.